src/Microsoft.AspNet.Mvc.TagHelpers/Internal/FileProviderGlobbingFile.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using Microsoft.AspNet.FileProviders;
+using Microsoft.Framework.FileSystemGlobbing.Abstractions;
+
+namespace Microsoft.AspNet.Mvc.TagHelpers.Internal
+{
+    public class FileProviderGlobbingFile : FileInfoBase
+    {
+        private readonly DirectoryInfoBase _parent;
```

Comment:
Since this is internal, is there anything wrong with making `_parent` of type `FileProviderGlobbingDirectory`